vocode/streaming/models/transcript.py

```diff
@@ -210,6 +211,13 @@ def to_string(self, include_timestamp: bool = False) -> str:
             return f"{self.sender.name}: {self.text} ({self.timestamp})"
         return f"{self.sender.name}: {self.text}"
 
-
 class TranscriptCompleteEvent(Event, type=EventType.TRANSCRIPT_COMPLETE):
     transcript: Transcript
+
+class JsonTranscriptEvent(Event, type=EventType.JSON_TRANSCRIPT):
+    transcript: StateAgentTranscript
+    def __init__(
+        self,
+        json_transcript: StateAgentTranscript
```

Comment:
This is wrong don't overide the init in pydantic you are going to mess up the validation, prefer class factory functions if you have alternate behavior you want, also you didn't  call super be careful about this when you are overiding init and you are inheriting from a library which is designed around the construction of classes (metaclasses). 
. Under the hood pydantic has a c  ffi in v2 (and bindings to the rust core) whereas the __init__ function is only for python init. 
I can't tell you all the ways this will break in the future, but it will be a less of headache down the line if we stick to the intended usage of the library.

Reply:
thanks for catching. Yeah we shouldn't need the init, it looks like other events don't need it